logprep/util/helper.py

```diff
@@ -41,18 +41,23 @@ def add_field_to(event, output_field, content, extends_lists=False):
         Original log-event that logprep is currently processing
     output_field: str
         Dotted subfield string indicating the target of the output value, e.g. destination.ip
-    content: str, dict
+    content: str, list, dict
         Value that should be written into the output_field, can be a str or dict object
```

Comment:
Value that should be written into the output_field, can be a str, list or dict object